src/features/instance/useProcessNext.tsx

```diff
@@ -78,6 +80,7 @@ export function useProcessNext() {
     },
     onError: (error: HttpClientError) => {
       window.logError('Process next failed:\n', error);
+      displayError(error);
```

Comment:
Could it work with just throwing the error instead? Alternatively, throw errors in the render function when they are returned from the query. This way it would be caught by the error boundary, and you wouldn't need the provider.

Reply:
Hmm, yeah, that's right. :thinking: I guess that's a downside with component-level error boundaries. Setting this as resolved.